common4j/src/main/com/microsoft/identity/common/java/providers/nativeauth/NativeAuthConstants.kt

```diff
@@ -0,0 +1,11 @@
+package com.microsoft.identity.common.java.providers.nativeauth
+
+object NativeAuthConstants {
+    object GrantType {
+        const val PASSWORD = "password"
+        const val OOB = "oob"
+        const val SLT = "slt"
+        const val REDIRECT = "redirect"
+        const val ATTRIBUTES = "attributes"
+    }
+}
```

Comment:
Nit: New line.

I'm going to stop commenting about this, Let's just check each file and make sure an new line ending is present

Reply:
I have checked all files and added newline at the end of the file